Gemfile.lock

```diff
@@ -32,6 +32,7 @@ GEM
       json (~> 1.4)
```

Comment:
Last thing, but could you please not commit this file either? It will mess up the deployment with all of the windows-specific things in there.